src/index.js

```diff
@@ -407,12 +408,14 @@ class CopyPlugin {
     logger.log(`begin globbing '${glob}'...`);
 
     /**
-     * @type {GlobEntry[]}
+     * @type {string[]}
      */
     let globEntries;
 
     try {
-      globEntries = await globby(glob, globOptions);
+      globEntries = globOptions.onlyDirectories
```

Comment:
Let's avoid it and just add the `onlyDirectories` doesn't supported in documentation. It doesn't make sense for our plugin anyway.

And we will still make a major release after this pr